src/wui/plot_area.cc

```diff
@@ -231,6 +231,11 @@ void draw_diagram(uint32_t time_ms,
 	// Make sure we haven't more ticks than we have space for -> avoid overlap
 	how_many_ticks = std::min(how_many_ticks, calc_plot_x_max_ticks(inner_w));
 
+	// Make sure how_many_ticks is a divisor of max_x
+	while (max_x % how_many_ticks != 0) {
```

Comment:
This should be more efficient:

```
if (max_x % how_many_ticks != 0) {
    how_many_ticks -= max_x % how_many_ticks;
}
```

Reply:
No, this is wrong. Notice we're decreasing the divisor, not the dividend.

Reply:
I don't have my math hat on right now - is there no solution that avoids having to have a loop?

Reply:
I can't think of a simple way, and I see no need for it either. This is not performance critical code, shaving off a few machine instructions will make no difference.

Reply:
OK, let's keep it like it is then.

Reply:
OK, let's have it then. Travis failure is just the usual timeout, so I'll merge this.